impl/src/main/java/org/ehcache/impl/internal/DefaultTimeSourceService.java

```diff
@@ -33,7 +33,7 @@ public DefaultTimeSourceService(TimeSourceConfiguration config) {
     if (config != null) {
       timeSource = config.getTimeSource();
     } else {
-      timeSource = SystemTimeSource.INSTANCE;
+      timeSource = new TickingTimeSource();
```

Comment:
I'd much rather this be the other way round, and have the default behavior be the current timesource.

Reply:
So be it. I will write a performance documentation page about it.

Reply:
I'd talk this over with others (say Chris G & Ludovic) before going to this effort... make sure that we have consensus.